server/index.js

```diff
@@ -25,6 +25,30 @@ app.get('/api/products', (req, res) => {
     });
 });
 
+app.get('/api/products/:product_id/styles', (req, res) => {
```

Comment:
I also wrote get request for product styles, and Run also had a merge conflict last meeting with his server file. I am worried we will end up constantly having merge conflicts with our server files because we are individually adding different http requests, some of which may overlap. Perhaps we can get together and consolidate our server file.

Reply:
I agree, we definitely should consolidate. Is yours much different from this one?

Reply:
It's not exactly the same, we can consolidate in a stand up meeting.